boot.php

```diff
@@ -654,39 +655,44 @@ function __construct() {
 
 
 		$this->scheme = 'http';
-		if((x($_SERVER,'HTTPS') && $_SERVER['HTTPS']) ||
+		if ((x($_SERVER,'HTTPS') && $_SERVER['HTTPS']) ||
 		   (x($_SERVER['HTTP_FORWARDED']) && preg_match("/proto=https/", $_SERVER['HTTP_FORWARDED'])) ||
 		   (x($_SERVER['HTTP_X_FORWARDED_PROTO']) && $_SERVER['HTTP_X_FORWARDED_PROTO'] == 'https') ||
 		   (x($_SERVER['HTTP_X_FORWARDED_SSL']) && $_SERVER['HTTP_X_FORWARDED_SSL'] == 'on') ||
 		   (x($_SERVER['FRONT_END_HTTPS']) && $_SERVER['FRONT_END_HTTPS'] == 'on') ||
 		   (x($_SERVER,'SERVER_PORT') && (intval($_SERVER['SERVER_PORT']) == 443)) // XXX: reasonable assumption, but isn't this hardcoding too much?
 		   ) {
 			$this->scheme = 'https';
-		   }
+		}
```

Comment:
I agree with your CR comment that `killme()` is no good in background processes. So I remove mine.